utils/highlight.ts

```diff
@@ -21,7 +21,7 @@ export const trimPreview = (preview: string) =>
 /**
  * Traverses a search item, returning highlighted HTML.
  */
-const getHighlight = ({ search, title, description, content }: IHighlightProps) => {
+export const getHighlight = ({ search, title, description, content }: IHighlightProps) => {
```

Comment:
Might want to make this more modular and inline some of this logic into SearchItem.